src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -260,7 +259,9 @@ private EmailWrapper generateFeedbackSessionEmailBaseForInstructorReminders(
                 "${submitUrl}", submitUrl,
                 "${reportUrl}", reportUrl,
                 "${coOwnersEmails}", coOwnersLine,
-                "${supportEmail}", Config.SUPPORT_EMAIL);
+                "${supportEmail}", Config.SUPPORT_EMAIL,
+                "${additionalInstructions}", additionalInstructions,
+                "${feedbackAction}", feedbackAction);
```

Comment:
`additionalInstructions` can be replaced with `""`, `feedbackAction` can be replaced with `Const.EmailTemplateMessages.FEEDBACK_ACTION_SUBMIT`.